web/thesauruses/javascript/ECMAScript 2009/io.json

```diff
@@ -0,0 +1,357 @@
+{
+  "meta": {
+    "language": "javascript",
+    "language_version": "ECMAScript 2009",
+    "language_name": "JavaScript",
+    "structure": "io"
+  },
+  "concepts": {
+    "write_line": {
+      "name": "Write a line to the console",
+      "code": "console.log(content);"
+    },
+    "write_line_with_new_line": {
+      "name": "Write a line to the console with a return character",
+      "code": "console.log(content + \"\\n\");"
```

Comment:
I think those should be `process.stdout.write(content)` and `console.log(content)` respectively because `console.log()` will append a newline at the end.

Reply:
You are right, however `process` is specific to Node.js and not JavaScript. I can always add this precision in `comment`.

Reply:
I have updated the first section of the IO concept by adding process.stdout.write as an alternative to console.log. Is there something else to add or edit?